### PR TITLE
Avatar alert

### PR DESCRIPTION
**Pull request by pankaj-prog**
demo for two comp.

### CODE REVIEW

**Comment by pankaj-prog** [comment]
https://deploy-preview-2--ui-library-demo.netlify.app/demo.html 
Preview link

File: index.html [hunk 1]
```diff
@@ -4,7 +4,14 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
     <title>UI Library</title>
   </head>
-  <body></body>
```
Comment (aiman-ansari) [inline index.html:9]: You have written body tag two times.

**Review by aiman-ansari — APPROVED** [review]
Everything is good you have given meaningful names to css class.

File: index.html (hunk 1 above)
Comment (pankaj-prog) [reply index.html:9]: Thankyou for reviewing. I will remove that

**Comment by deekshithmd** [comment]
code looks clean and file structure also looks good

**Review by manojsarna — COMMENTED** [review]
Code looks fine 

try to add more examples. 

add code code snippets

**Pull request merged**